daemon/execdriver/windows/exec.go

```diff
@@ -34,21 +33,11 @@ func (d *Driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessCo
 	// Configure the environment for the process // Note NOT c.ProcessConfig.Tty
 	createProcessParms.Environment = setupEnvironmentVariables(processConfig.Env)
 
-	// While this should get caught earlier, just in case, validate that we
-	// have something to run.
-	if processConfig.Entrypoint == "" {
-		err = errors.New("No entrypoint specified")
-		logrus.Error(err)
-		return -1, err
-	}
+	createProcessParms.CommandLine, err = createCommandLine(&c.ProcessConfig, !c.ArgsEscaped)
```

Comment:
Can we not revert argument here, but use `!` inside funtion? I think it's sorta hard to read.